melodic/distribution.yaml

```diff
@@ -8072,6 +8072,21 @@ repositories:
       url: https://github.com/CPFL/robosense.git
       version: develop-curves-function
     status: developed
+  robosense_simulator:
```

Comment:
You lost the doc section compared to #24293 I think it would make sense to add that back.